content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -317,12 +318,12 @@ This code set may include Marital Statuses such as Divorced and Married.
 
 ##### Code Set 40 Person Relationships
 
-This code set is used to describe the patient's relationship to Subscriber, Guarantor, Emergency Contact, Next of Kin, and other related persons.
+This code set is used to describe the related person's relationship to the patient
```

Comment:
why did this description change?

Reply:
because we switched the maping from relation to inverse relation on the local service to represent the relatedperson relationship to the patient instead of the patients relationship to the related person. Things that were displaying as child before are now displaying as mother, etc.